marketplace_standard_app_api/models/object_storage.py

```diff
@@ -36,3 +36,17 @@ class DatasetModel(BaseModel):
 
 
 DatasetListResponse = List[DatasetModel]
+
+
+class MappingName(ConstrainedStr):
+    min_length = 1
+    max_length = 255
+
+
+class MappingModel(BaseModel):
+    name: MappingName
+    type: str
```

Comment:
What exactly does the `type` indicate?

Reply:
This is not required anymore after [our discussion](https://github.com/materials-marketplace/standard-app-api/pull/34#issuecomment-1178932599). The schema type will be send in the headers with the data.